2025-01-29 13:13:37.046 [info] (Client) Pylance async client (2024.12.1) started with python extension (2024.22.2)
2025-01-29 13:13:37.310 [info] [Info  - 1:13:37 PM] (3108) Server root directory: file:///home/dewain/.vscode-server/extensions/ms-python.vscode-pylance-2024.12.1/dist
2025-01-29 13:13:37.310 [info] [Info  - 1:13:37 PM] (3108) Pylance language server 2024.12.1 (pyright version 1.1.389, commit ce1325fc) starting
2025-01-29 13:14:27.434 [info] [Info  - 1:14:27 PM] (3108) Starting service instance "<default>"
2025-01-29 13:14:27.452 [info] [Info  - 1:14:27 PM] (3108) Setting environmentName for service "<default>": "3.12.3 (global)"
2025-01-29 13:14:27.452 [info] [Info  - 1:14:27 PM] (3108) Setting pythonPath for service "<default>": "/bin/python3"
2025-01-29 13:14:27.453 [info] [Info  - 1:14:27 PM] (3108) No include entries specified; assuming /<default workspace root>
2025-01-29 13:14:27.453 [info] [Info  - 1:14:27 PM] (3108) Auto-excluding **/node_modules
2025-01-29 13:14:27.453 [info] [Info  - 1:14:27 PM] (3108) Auto-excluding **/__pycache__
2025-01-29 13:14:27.453 [info] [Info  - 1:14:27 PM] (3108) Auto-excluding **/.*
2025-01-29 13:14:27.471 [info] [Info  - 1:14:27 PM] (3108) Assuming Python version 3.12.3.final.0
2025-01-29 13:14:27.510 [info] [Error - 1:14:27 PM] (3108) File or directory "/<default workspace root>" does not exist.
2025-01-29 13:14:27.511 [info] [Info  - 1:14:27 PM] (3108) No source files found.
2025-01-29 13:14:27.708 [info] [Info  - 1:14:27 PM] (3108) Background analysis(1) root directory: file:///home/dewain/.vscode-server/extensions/ms-python.vscode-pylance-2024.12.1/dist
2025-01-29 13:14:27.709 [info] [Info  - 1:14:27 PM] (3108) Background analysis(1) started
